test/polymer_gestures_test.html

```diff
@@ -0,0 +1,218 @@
+<!--
+Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+for details. All rights reserved. Use of this source code is governed by a
+BSD-style license that can be found in the LICENSE file.
+-->
+<!doctype html>
+<html>
+  <head>
+    <script src="packages/web_components/webcomponents-lite.js"></script>
+    <link rel="import" href="packages/web_components/interop_support.html">
+    <link rel='import' href='packages/polymer_interop/polymer.html'>
+
+    <link rel='x-dart-test' href='polymer_gestures_test.dart'>
+    <!--<script type="application/dart" src='polymer_base_test.dart'></script>-->
+    <!--<script src="packages/browser/dart.js"></script>-->
+    <script src='packages/test/dart.js'></script>
+  </head>
+  <body>
+
+  <dom-module id="x-foo">
+    <style>
+      #div {
+        height: 40px;
+        background: red;
+      }
+    </style>
+
+    <template>
+      <div id="div"></div>
+    </template>
+
+    <script>
+      Polymer({
+        is: 'x-foo',
+        listeners: {
+          tap: 'tapHandler'
+        },
+        tapHandler: function(e) {
+          var ev = Polymer.dom(e);
+          this._testLocalTarget = ev.localTarget;
+          this._testRootTarget = ev.rootTarget;
+        }
+      });
+    </script>
+  </dom-module>
+
+  <dom-module id="x-app">
+
+    <template>
+      <x-foo id="foo"></x-foo>
+    </template>
+
+    <script>
+      Polymer({
```

Comment:
Oh shoot, I forgot this was polymer_interop. In that case you can leave these as is.